src/ProjectNeon/Assets/Features/Battle/Variables/SpellHitsLastCardMultiplier.cs

```diff
@@ -0,0 +1,11 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class SpellHitsLastCardMultiplier : FloatVariable
+{
+    [SerializeField] CardResolutionZone resolutionZone;
```

Comment:
Number variables should not have any external dependencies.

Reply:
@SilasReinagel But then, how could I acces this value from the unity editor? How could I project this value onto something if I can't makeit visible from insie this variable?